ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/questionMark.html

```diff
@@ -1,4 +1,4 @@
 <template data-sly-template.questionMark="${@ componentId, longDescription, bemBlock}">
-  <button class="${bemBlock}__questionmark" data-sly-test="${longDescription}" title="Help text">
+  <button class="${bemBlock}__questionmark" style="${longDescription ? '' : 'display:none;'} title="Help text">
```

Comment:
styles need to come via theme, code should not add inline style, it would override other styles provided by customer. This looks incorrect to me